### DISCUSSION TITLE
Character encoding in getTitle()

### DISCUSSION THREAD

**Discussion by chungddong** [s-frei/TrackSearch#198] [Q&A] [answered]
First of all, thank you for making a really good library. I am a user in Korea and I am using this library.
When I use getTitle() to display the title of the search result, languages like English are displayed normally, but Korean results are not displayed normally. 
Is there any way to set the text encoding separately?

**Comment by s-frei** [comment]
Hi @chungddong nice to here that it is used across the globe 😉 
Do you have a link and an expected result for me to dig into this?

**Reply by chungddong** [reply]
Yes, I am trying to get the result using the following code.

`YouTubeTrack yTrack = explicitClient.getTrack("https://www.youtube.com/watch?v=cFgk2PMgPJ4");`
`System.out.println("Title : " + yTrack.getTitle());`

Currently, the result looks like 'Title : ?쑄?븯(YOUNHA) - ?삤瑜댄듈援щ쫫 [媛??궗/Lyrics]'.

I want the expected result to be something like 'Title : 윤하(YOUNHA) - 오르트구름 [가사/Lyrics] '.

**Reply by s-frei** [reply, marked as answer]
Well, got a little flashback as there was already an issue for something pretty similar: https://github.com/s-frei/TrackSearch/issues/139

I think you should handle the decoding on your side as the one reported that issue or maybe force your VM to use UTF-8 using the option:
```shell
-Dfile.encoding=UTF-8
```

**Reply by chungddong** [reply]
Resolved. Thank you!
